# This file is distributed under the same license as the Django package.
#
# Translators:
# Jannis Leidel <jannis@leidel.info>, 2011
# arupakan125 <koh@arupaka.net>, 2020
# Masashi SHIBATA <contact@c-bata.link>, 2017
# Nikita K <hiyori.amatsuki@gmail.com>, 2019
# Shinya Okano <tokibito@gmail.com>, 2013-2016,2021,2023
msgid ""
msgstr ""
"Project-Id-Version: django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-01-14 20:40+0800\n"
"PO-Revision-Date: 2023-04-25 08:09+0000\n"
"Last-Translator: Shinya Okano <tokibito@gmail.com>, 2013-2016,2021,2023\n"
"Language-Team: Japanese (http://www.transifex.com/django/django/language/"
"ja/)\n"
"Language: ja\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: .\venv\Lib\site-packages\django\contrib\auth\admin.py:49
msgid "Personal info"
msgstr "個人情報"

#: .\venv\Lib\site-packages\django\contrib\auth\admin.py:51
msgid "Permissions"
msgstr "パーミッション"

#: .\venv\Lib\site-packages\django\contrib\auth\admin.py:62
msgid "Important dates"
msgstr "重要な日程"

#: .\venv\Lib\site-packages\django\contrib\auth\admin.py:158
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr "主キーが %(key)r である %(name)s オブジェクトは存在しません。"

#: .\venv\Lib\site-packages\django\contrib\auth\admin.py:170
msgid "Password changed successfully."
msgstr "パスワードを変更しました"

#: .\venv\Lib\site-packages\django\contrib\auth\admin.py:191
#, python-format
msgid "Change password: %s"
msgstr "パスワードの変更: %s"

#: .\venv\Lib\site-packages\django\contrib\auth\apps.py:16
msgid "Authentication and Authorization"
msgstr "認証と認可"

#: .\venv\Lib\site-packages\django\contrib\auth\base_user.py:59
msgid "password"
msgstr "パスワード"

#: .\venv\Lib\site-packages\django\contrib\auth\base_user.py:60
msgid "last login"
msgstr "最終ログイン"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:41
msgid "No password set."
msgstr "パスワードは設定されませんでした。"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:49
msgid "Invalid password format or unknown hashing algorithm."
msgstr "無効なパスワードか不明なハッシュアルゴリズムです。"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:99
#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:393
#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:471
msgid "The two password fields didn’t match."
msgstr "確認用パスワードが一致しません。"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:102
#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:180
#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:215
#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:475
msgid "Password"
msgstr "パスワード"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:108
msgid "Password confirmation"
msgstr "パスワード(確認用)"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:111
#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:486
msgid "Enter the same password as before, for verification."
msgstr "確認のため、再度パスワードを入力してください。"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:182
msgid ""
"Raw passwords are not stored, so there is no way to see this user’s "
"password, but you can change the password using <a href=\"{}\">this form</a>."
msgstr ""
"生のパスワードは格納されていないため、このユーザのパスワードを確認する方法は"
"ありません。しかし<a href=\"{}\">このフォーム</a>を使用してパスワードを変更で"
"きます。"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:222
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr ""
"正しい%(username)sとパスワードを入力してください。どちらのフィールドも大文字"
"と小文字は区別されます。"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:225
msgid "This account is inactive."
msgstr "アカウントが無効です。"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:290
msgid "Email"
msgstr "メールアドレス"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:396
msgid "New password"
msgstr "新しいパスワード"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:402
msgid "New password confirmation"
msgstr "新しいパスワード(確認用)"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:439
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr "元のパスワードが間違っています。もう一度入力してください。"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:443
msgid "Old password"
msgstr "元のパスワード"

#: .\venv\Lib\site-packages\django\contrib\auth\forms.py:483
msgid "Password (again)"
msgstr "パスワード(確認用)"

#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:344
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:437
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:527
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:622
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:682
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:724
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:782
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:837
msgid "algorithm"
msgstr "アルゴリズム"

#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:345
msgid "iterations"
msgstr "イテレーション"

#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:346
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:443
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:529
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:626
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:683
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:725
msgid "salt"
msgstr "ソルト"

#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:347
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:444
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:627
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:684
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:726
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:783
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:838
msgid "hash"
msgstr "ハッシュ"

#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:438
msgid "variety"
msgstr "多様性"

#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:439
msgid "version"
msgstr "バージョン"

#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:440
msgid "memory cost"
msgstr "メモリーコスト"

#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:441
msgid "time cost"
msgstr "時間コスト"

#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:442
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:625
msgid "parallelism"
msgstr "並行性"

#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:528
#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:623
msgid "work factor"
msgstr "ワークファクター"

#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:530
msgid "checksum"
msgstr "チェックサム"

#: .\venv\Lib\site-packages\django\contrib\auth\hashers.py:624
msgid "block size"
msgstr "ブロックサイズ"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:62
#: .\venv\Lib\site-packages\django\contrib\auth\models.py:116
msgid "name"
msgstr "名前"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:66
msgid "content type"
msgstr "コンテンツタイプ"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:68
msgid "codename"
msgstr "コード名"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:73
msgid "permission"
msgstr "パーミッション"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:74
#: .\venv\Lib\site-packages\django\contrib\auth\models.py:119
msgid "permissions"
msgstr "パーミッション"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:126
msgid "group"
msgstr "グループ"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:127
#: .\venv\Lib\site-packages\django\contrib\auth\models.py:258
msgid "groups"
msgstr "グループ"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:249
msgid "superuser status"
msgstr "スーパーユーザー権限"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:252
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr "全ての権限を持っているとみなされます。"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:261
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr ""
"このユーザーが所属するグループ。ユーザーはそれぞれのグループに付与されたすべ"
"ての権限を持ちます。"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:269
msgid "user permissions"
msgstr "ユーザーパーミッション"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:271
msgid "Specific permissions for this user."
msgstr "このユーザーの持つ権限です。"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:345
msgid "username"
msgstr "ユーザー名"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:349
msgid "Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr ""
"この項目は必須です。半角アルファベット、半角数字、@/./+/-/_ で150文字以下にし"
"てください。"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:353
msgid "A user with that username already exists."
msgstr "同じユーザー名が既に登録済みです。"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:356
msgid "first name"
msgstr "名"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:357
msgid "last name"
msgstr "姓"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:358
msgid "email address"
msgstr "メールアドレス"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:360
msgid "staff status"
msgstr "スタッフ権限"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:362
msgid "Designates whether the user can log into this admin site."
msgstr "ユーザーが管理サイトにログイン可能かどうかを示します。"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:365
msgid "active"
msgstr "有効"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:368
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""
"ユーザーがアクティブかどうかを示します。アカウントを削除する代わりに選択を解"
"除してください。"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:372
msgid "date joined"
msgstr "登録日"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:381
msgid "user"
msgstr "ユーザー"

#: .\venv\Lib\site-packages\django\contrib\auth\models.py:382
msgid "users"
msgstr "ユーザー"

#: .\venv\Lib\site-packages\django\contrib\auth\password_validation.py:111
#, python-format
msgid ""
"This password is too short. It must contain at least %(min_length)d "
"character."
msgid_plural ""
"This password is too short. It must contain at least %(min_length)d "
"characters."
msgstr[0] "このパスワードは短すぎます。最低 %(min_length)d 文字以上必要です。"

#: .\venv\Lib\site-packages\django\contrib\auth\password_validation.py:123
#, python-format
msgid "Your password must contain at least %(min_length)d character."
msgid_plural "Your password must contain at least %(min_length)d characters."
msgstr[0] "パスワードは最低 %(min_length)d 文字以上必要です。"

#: .\venv\Lib\site-packages\django\contrib\auth\password_validation.py:206
#, python-format
msgid "The password is too similar to the %(verbose_name)s."
msgstr "このパスワードは %(verbose_name)s と似すぎています。"

#: .\venv\Lib\site-packages\django\contrib\auth\password_validation.py:213
msgid "Your password can’t be too similar to your other personal information."
msgstr "あなたの他の個人情報と似ているパスワードにはできません。"

#: .\venv\Lib\site-packages\django\contrib\auth\password_validation.py:245
msgid "This password is too common."
msgstr "このパスワードは一般的すぎます。"

#: .\venv\Lib\site-packages\django\contrib\auth\password_validation.py:250
msgid "Your password can’t be a commonly used password."
msgstr "よく使われるパスワードにはできません。"

#: .\venv\Lib\site-packages\django\contrib\auth\password_validation.py:261
msgid "This password is entirely numeric."
msgstr "このパスワードは数字しか使われていません。"

#: .\venv\Lib\site-packages\django\contrib\auth\password_validation.py:266
msgid "Your password can’t be entirely numeric."
msgstr "数字だけのパスワードにはできません。"

#: .\venv\Lib\site-packages\django\contrib\auth\templates\registration\password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr "%(site_name)s のパスワードリセット"

#: .\venv\Lib\site-packages\django\contrib\auth\validators.py:12
msgid ""
"Enter a valid username. This value may contain only unaccented lowercase a-z "
"and uppercase A-Z letters, numbers, and @/./+/-/_ characters."
msgstr ""
"有効なユーザー名を入力してください。アクセント記号を除いた半角英数、 @/./+/-/"
"_ を使用できます。"

#: .\venv\Lib\site-packages\django\contrib\auth\validators.py:22
msgid ""
"Enter a valid username. This value may contain only letters, numbers, and "
"@/./+/-/_ characters."
msgstr ""
"有効なユーザー名を入力してください。半角英数、 @/./+/-/_ を使用できます。"

#: .\venv\Lib\site-packages\django\contrib\auth\views.py:163
msgid "Logged out"
msgstr "ログアウト"

#: .\venv\Lib\site-packages\django\contrib\auth\views.py:222
msgid "Password reset"
msgstr "パスワードをリセット"

#: .\venv\Lib\site-packages\django\contrib\auth\views.py:249
msgid "Password reset sent"
msgstr "パスワードリセットを送信しました"

#: .\venv\Lib\site-packages\django\contrib\auth\views.py:259
msgid "Enter new password"
msgstr "新しいパスワードを入力してください"

#: .\venv\Lib\site-packages\django\contrib\auth\views.py:331
msgid "Password reset unsuccessful"
msgstr "パスワードのリセットに失敗しました"

#: .\venv\Lib\site-packages\django\contrib\auth\views.py:340
msgid "Password reset complete"
msgstr "パスワードがリセットされました"

#: .\venv\Lib\site-packages\django\contrib\auth\views.py:352
msgid "Password change"
msgstr "パスワードの変更"

#: .\venv\Lib\site-packages\django\contrib\auth\views.py:375
msgid "Password change successful"
msgstr "パスワードを変更しました"
